docs/checks/duplicateMapFeatureCheck.md

```diff
@@ -2,20 +2,20 @@
 
 #### Description
 The DuplicateMapFeaturecheck flags node, way or relation which have duplicate map features in areas or connected locations.
-[Osmose#4080](https://wiki.openstreetmap.org/wiki/Osmose/issues#4080):
-    Details
-    Object tagged twice as node, way or relation:
-    Class 1 "Object tagged twice as node and way" 
-    Class 2 "Object tagged twice as way and relation" 
-    Class 3 "Object tagged twice as node and relation" 
+[Osmose#4080](https://wiki.openstreetmap.org/wiki/Osmose/issues#4080): \
```

Comment:
This \ is showing up when rendered.
![Screen Shot 2021-09-07 at 8 48 44 AM](https://user-images.githubusercontent.com/24948563/132374404-bdf33061-43b5-43a3-b4c0-f02fd484673d.png)

Reply:
thanks Bentley, updated it